app/domain/NotificationsForEori.scala

```diff
@@ -17,16 +17,29 @@
 package domain
 
 import models.EORI
-import org.joda.time.DateTime
 import play.api.libs.json._
-import uk.gov.hmrc.mongo.play.json.formats.MongoJodaFormats
+
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 
 case class NotificationsForEori(eori: EORI,
                                 notifications: Seq[Notification],
-                                lastUpdated: Option[DateTime])
+                                lastUpdated: Option[LocalDateTime])
 
 object NotificationsForEori {
-  implicit val lastUpdatedFormat: Format[DateTime] = MongoJodaFormats.dateTimeFormat
+
+  implicit val lastUpdatedDateTimeFormat: Format[LocalDateTime] = Format[LocalDateTime](
+    Reads[LocalDateTime](js =>
+      js.validate[Long] match {
+        case JsSuccess(epoc, _) => JsSuccess(Instant.ofEpochMilli(epoc).atOffset(ZoneOffset.UTC).toLocalDateTime)
+        case _ =>
```

Comment:
I don't think the _ case needs to be on a different line? especially when the first case isn't and it is longer